### PR TITLE
WIP: add nose

### PR DESCRIPTION
**Pull request by msarahan**
This one might take some polish.  It does some strange things to remove the setup.py entry point stuff.

### CODE REVIEW

File: recipes/nose/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+test:
+  commands:
+    - nosetests -h
+  imports:
+    - nose
+
+about:
+  home: http://readthedocs.org/docs/nose/
+  license: LGPL
```
Comment (ocefpaf) [inline recipes/nose/meta.yaml]: It is `LGPL-2.0`.

File: recipes/nose/bld.bat [hunk 2]
```diff
@@ -0,0 +1,2 @@
+python setup.py install
```
Comment (jakirkham) [inline recipes/nose/bld.bat]: If we are using `setuptools`, we need to add the following arguments.

```
--single-version-externally-managed --record record.txt
```

File: recipes/nose/build.sh [hunk 3]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+$PYTHON setup.py install
```
Comment (jakirkham) [inline recipes/nose/build.sh]: Same as above.

File: recipes/nose/build.sh [hunk 4]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+$PYTHON setup.py install --single-version-externally-managed --record=record.txt
+
+rm -rf $PREFIX/man
+rm -rf $SP_DIR/man
+rm -rf $SP_DIR/nose-*.egg/man
+rm -f $PREFIX/bin/nosetests-*
```
Comment (jakirkham) [inline recipes/nose/build.sh]: What are these?

Comment (ocefpaf) [inline recipes/nose/build.sh]: That is the same script as `nosetests` but with the python tag (`nosetest-2.7`). In conda's model it is useless, but if we one day can install two different Python versions in one `env`, then that is handy.

**Pull request merged**